Messenger/MessengerJsonRpc.cpp

```diff
@@ -75,7 +75,7 @@ namespace Plugin {
         const string& room = params.Room.Value();
 
         if (!user.empty() && !room.empty()) {
-            string roomId = JoinRoom(room, user);
+            string roomId = JoinRoom((room + "_" + string(params.Secure.Data())), user);
```

Comment:
The roomId is still postfixed with an '_' and called "insecure" or "secure". This, in my opinion is causing a very confusing impleentation (what if users start using the '_' and start using insecure/secure as postfix ? I thought we discussed to make this part of the map in which it is stored. Will indicate later where.
As our customers do not make use of this feature will mark this as a minor which means it is good to merge, although it might be a good idea to later on update this to an "explicit" flag that is *not* partof the room id.